src/DssSpell.sol

```diff
@@ -54,32 +63,37 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     //
 
     function officeHours() public override returns (bool) {
-        return true;
+        return false;
     }
 
     function actions() public override {
 
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        onboardNewCollaterals();
+        // onboardNewCollaterals();
         // offboardCollaterals();
 
-        drawFromRWA009Urn();
+        address RWA009_A_URN = DssExecLib.getChangelogAddress("RWA009_A_URN");
+        address MCD_VEST_DAI = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
 
-        // Add RWA_TOKEN_FAB to changelog
-        DssExecLib.setChangelogAddress("RWA_TOKEN_FAB", RWA_TOKEN_FAB);
-
-        DssExecLib.setChangelogVersion("1.13.3");
-    }
+        // Huntingdon Valley (HVBank) Vault Drawdown
+        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
 
-    function drawFromRWA009Urn() internal {
-        // lock RWA009 Token in the URN
-        ERC20Like(RWA009).approve(RWA009_A_URN, 1 * WAD);
-        RwaUrnLike(RWA009_A_URN).lock(1 * WAD);
 
-        // draw DAI to genesis address
-        RwaUrnLike(RWA009_A_URN).draw(RWA009_DRAW_AMOUNT);
+        // Keep3r Network Stream Re-Deployment
+        VestLike(MCD_VEST_DAI).yank(8);
+        VestLike(MCD_VEST_DAI).restrict(
+            VestLike(MCD_VEST_DAI).create({
+                usr: 0x37b375e3D418fbECba6b283e704F840AB32f3b3C,
```

Comment:
I'm wondering if we should add this address to the changelog. It's been provided to us as part of the MIP process and has some `constant` values in it related to things like the vest contract and the vow, which may change in the future. We might want to track this so that when we have dependent contracts handy for upgrade planning.

Reply:
Maybe unnecessary since we're tracking it in addresses_wallets.sol.

Reply:
Yeah, I'd vote not to add to the chainlog for now as we don't have any other vesting address.